lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -153,36 +167,19 @@ function _campaign_resource_access($op = 'view', $args = array()) {
 /**
  * Callback for Campaigns index.
  */
-function _campaign_resource_index($parameters) {
-  // Initialize output.
-  $index = array();
-
-  // Get the results of the explore_campaigns view.
-  $view = views_get_view('explore_campaigns');
-  $view->execute();
-
-  $filter_staff_pick = FALSE;
-  if (isset($parameters['is_staff_pick']) && $parameters['is_staff_pick']) {
-    $filter_staff_pick = TRUE;
+function _campaign_resource_index($is_staff_pick = NULL, $on_mobile_app = NULL, $term_ids = NULL) {
```

Comment:
If turning `is_staff_pick` & `on_mobile_app` to a booleans, then switch these to default to `false` :)